virttest/ssh_key.py

```diff
@@ -232,7 +234,7 @@ def setup_remote_known_hosts_file(client_ip, server_ip,
 
     cmd = "%s %s" % (abs_path, client_ip)
     host_key = decode_to_text(process.system_output(cmd, verbose=False))
-    remote_known_hosts_file = remote.RemoteFile(
+    remote_known_hosts_file = remote_old.RemoteFile(
```

Comment:
why don't we consider `RemoteFile()` and `RemoteRunner()` to be moved to aexpect ?

Reply:
Hi @balamuruhans, these:

1) involve additional dependencies that are specific to avocado-vt
2) are rarely used and extra complicated
3) duplicate most of the core functionality already provided by remote and thus aexpect

For more details and sources for all these three conclusions, please see [the analogous discussion on the aexpect side](https://github.com/avocado-framework/aexpect/pull/55#issuecomment-493401973).